src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -11,6 +11,14 @@ namespace Microsoft.CodeAnalysis.DocumentationComments
 {
     internal abstract class AbstractDocumentationCommentFormattingService : IDocumentationCommentFormattingService
     {
+        private enum DocumentationCommentListType
+        {
+            None,
+            Bullet,
+            Number,
+            Table,
```

Comment:
can you doc this (at least the 'Table' bit).  I'm curious how tables work here.

Reply:
Tables do not work. The primitives necessary to render a grid are not available.

Reply:
➡️ Filed follow-up issue #37506